src/helpers/constants/error.messages.js

```diff
@@ -23,5 +23,7 @@ export default {
   gender: 'The gender should only contain alphanumeric characters',
   phoneNumber: 'The phone number is required',
   bio: 'The bio is required',
-  rating: 'you can not go beyond 5 as a rating number, Thanks'
+  rating: 'you can not go beyond 5 as a rating number, Thanks',
+  reason: 'this can not to be a string, Thanks',
```

Comment:
I think there is an issue here @MCFrank16

Reply:
which one?? @truestbyheart

Reply:
@MCFrank16 `tagList` shouldn't be in your PR